src/promptflow/promptflow/executor/_dag_manager.py

```diff
@@ -107,6 +113,7 @@ def _is_node_bypassable(self, node: Node) -> bool:
             # This is not a good practice, but we need to update the default output of bypassed node
             # to completed_nodes_outputs. We will remove these after skip config is deprecated.
             self.complete_nodes({node.name: skip_return})
+            self._bypassed_by_skip_config.append(node.name)
```

Comment:
Why should we add this? it is not that easy to understand?

Reply:
Removed.